.github/workflows/release-step-3.yml

```
@@ -221,16 +221,20 @@ jobs:
         uses: ./.github/workflows/gradle-goal
         with:
           command: ""
+      - name: Docs builder setup
+        uses: elastic/docs-actions/docs-builder/setup@v1
       - name: Print Release Notes
         id: print_release_notes
-        # note: release notes here will be copied as-is from 'CHANGELOG.next-release.md'
-        # the 'pre-post-release' workflow executed after this will reset contents of 'CHANGELOG.next-release.md'
+        # FIXME using temporary workaround from https://github.com/elastic/docs-builder/issues/3057
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:30: Double quote to prevent globbing and word splitting

(shellcheck)